ui/@build/rollupProject/index.js

```diff
@@ -17,6 +17,7 @@ module.exports = targets => {
               name: target.name,
               plugins: [
                 terser({
+		  safari10: true,
```

Comment:
This is not indented correctly.

Reply:
My bad, my editor was not well configured and did not expand tabs. Fixed now.